src/detect-http-raw-header.c

```diff
@@ -105,6 +112,16 @@ void DetectHttpRawHeaderRegister(void)
             PrefilterMpmHttpHeaderRawResponseRegister, NULL, ALPROTO_HTTP1,
             0); /* progress handled in register */
 
+    DetectAppLayerInspectEngineRegister2("http_raw_header", ALPROTO_HTTP2, SIG_FLAG_TOSERVER,
+            HTTP2StateDataClient, DetectEngineInspectBufferGeneric, GetData2);
+    DetectAppLayerInspectEngineRegister2("http_raw_header", ALPROTO_HTTP2, SIG_FLAG_TOCLIENT,
+            HTTP2StateDataServer, DetectEngineInspectBufferGeneric, GetData2);
+
+    DetectAppLayerMpmRegister2("http_raw_header", SIG_FLAG_TOSERVER, 2,
+            PrefilterMpmHttp2HeaderRawRequestRegister, NULL, ALPROTO_HTTP2, HTTP2StateDataClient);
```

Comment:
I don't see the reason for not using `PrefilterGenericMpmRegister` here instead? The prefilter func below looks standard

Reply:
Great :-)